README.md

```diff
@@ -7,125 +11,131 @@ Skuber is a Scala client library for [Kubernetes](http://kubernetes.io). It prov
-val nginxService = Service("nginx")
-    .withSelector("app" -> "nginx")
-    .exposeOnNodePort(30001 -> 80) 
-
-// Some standard Akka implicits that are required by the skuber v2 client API
-import akka.actor.ActorSystem
-import akka.stream.ActorMaterializer
-implicit val system = ActorSystem()
-implicit val materializer = ActorMaterializer()
-implicit val dispatcher = system.dispatcher
-
-// Initialise skuber client
-val k8s = k8sInit
-
-// Create the service and the deployment on the Kubernetes cluster
-val createOnK8s = for {
-  svc <- k8s create nginxService
-  dep  <- k8s create nginxDeployment
-} yield (dep,svc)
-
-createOnK8s onComplete {
-  case Success(_) => System.out.println("Successfully created nginx deployment & service on Kubernetes cluster")
-  case Failure(ex) => System.err.println("Encountered exception trying to create resources on Kubernetes cluster: " + ex)
-}
-
-k8s.close // this prevents any more requests being sent by the client
-system.terminate // this closes the connection resources etc.
-```
+## Quick Start
 
-## Prerequisites
+Make sure [prerequisites](#prerequisites) are met. There are couple of quick ways to get started with Skuber:
 
-A Kubernetes cluster is needed at runtime. For local development purposes, minikube is recommended.
+### With [Ammonite-REPL](http://ammonite.io/#Ammonite-REPL) 
 
-You need Java 8 to run Skuber.
+Provides you with a configured client on startup. It is handy to use this for quick experiments. 
+- using bash
+    
+  ```bash
+  $ amm -p ./Quickstart.sc
+  ```
 
-## Release
+- from inside ammonite-repl:
+  ```scala
+  import $file.`Quickstart`, Quickstart._
+  ```
 
-You can use the latest release (for Scala 2.11 or 2.12) by adding to your build:
+  ###### Just handy shortcut to import skuber inside ammonite-repl:
 
-```sbt
-libraryDependencies += "io.skuber" %% "skuber" % "2.0.9"    
-```
+  ```
+  import $ivy.`io.skuber::skuber:2.0.9`, skuber._, skuber.json.format._
+  ```
 
-Meanwhile users of skuber v1 can continue to use the latest (and possibly final, with exception of important fixes) v1.x release, which is available only on Scala 2.11:
+### Copy-paste ready snippet
 
-```sbt
-libraryDependencies += "io.skuber" % "skuber_2.11" % "1.7.1"
-```
+  This example lists pods in `kube-system` namespace. It unsafely assumes that your kubeconfig file exists and kubernetes cluster is accessible:
 
-## Migrating to release v2
+  ```scala
+  import skuber._
+  import skuber.json.format._
+  import akka.actor.ActorSystem
+  import akka.stream.ActorMaterializer
+  import scala.util.{Success, Failure}
 
-If you have a Skuber client using release v1.x and want to move to the strategic version 2 release, then check out the [migration guide](docs/MIGRATION_1-to-2.md).
+  implicit val system = ActorSystem()
+  implicit val materializer = ActorMaterializer()
+  implicit val dispatcher = system.dispatcher
 
-## Building
+  val cfg: Configuration = Configuration.parseKubeconfigFile().get
```

Comment:
To keep it as simple as possible I'd prefer to use default configuration
`  val k8s = k8sInit()`